### DISCUSSION TITLE
【丁真语录】玩绝区零的十大狠人

### DISCUSSION THREAD

**Discussion by yqj666** [share121/inter-knot#1111] [其他]
最后一个真的喷不了
来支持下孩子的视频吧o(╥﹏╥)o
求求了(づ｡◕‿‿◕｡)づ
真的太惨了
BV1JdYeeuE1s
【【丁真语录】绝区零十大神金-哔哩哔哩】 https://b23.tv/szTMZUy
话说以后还能在这里投稿
![Videoframe_20240810_231837_com huawei himovie local_edit_1021127378475669](https://github.com/user-attachments/assets/b4d56b3a-e1db-4daa-93c1-4e610ad9c0e3)

**Comment by yqj666** [comment]
【【丁真语录】绝区零十大神金-哔哩哔哩】 https://b23.tv/szTMZUy

**Comment by lllliufeng** [comment]
益言丁真

**Comment by yqj666** [comment]
玩绝区零的十大狠人，最后一个真的喷不了
第一，上号不刮刮刮卡的
第二，每天喝咖啡领60电池攒着不用的
第三，拿信号余波和信号残响买原装母带
第四，兑换码奖励不领的
第五人格启动
第六，信赖对话选项乱点的
第七，在141杂货铺乱买的
第八，成功领养小黑猫
第九，不让十八号连续工作七天的
第十，体力药全用来刷30级驱动盘
